portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,18 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Apoorva Dhanala's Portfolio</h1>
+      <p>Hi! My name is Apoorva Dhanala and I am a rising junior at the University of Georgia studying Computer Science with an emphasis in Artificial Intelligence. At UGA, I am founder and president of UGA’s chapter of Girls Who Code where we work with local, underprivileged Athens-Clarke County schools where UGA students volunteer weekly hoping to expose students to coding while closing the gender gap in technology. I am also involved with the Georgia Recruitment Team where I represent the admissions office as an ambassador at panels and tours, Athens Prison Tutorial where I tutor prison inmates at the Athens-Clarke County Jail who lack their high school diplomas and help them improve their skills in order to pass their GED examination, and with the Honors Program Student Council where I work to help bring academic and social opportunities to students in the Honors Program. Outside of classes and club involvement, I am a CURO Honors Research scholar where I have been involved with Linguistics and Natural Language Processing research and most recently, Android App development work.  
```

Comment:
Could you break up the paragraphs into lines (no specific column limit, not 80ish is nice) to help with readability, please?